lua/null-ls/builtins/diagnostics/sqlfluff.lua

```diff
@@ -6,6 +6,10 @@ return helpers.make_builtin({
     meta = {
         url = "https://github.com/sqlfluff/sqlfluff",
         description = "A SQL linter and auto-formatter for Humans",
+        notes = {
+            "SQLFluff needs a mandatory `--dialect` argument. Use `extra_args` to add yours. `extra_args` can also be a function to build more sophisticated logic.",
+        },
+        usage = 'local sources = {\n\tnull_ls.builtins.diagnostics.sqlfluff.with({\n\t\textra_args = {"--dialect", "postgres"} -- change to your dialect \n\t}) \n}',
```

Comment:
```suggestion
        usage = [[
local sources = {
	null_ls.builtins.diagnostics.sqlfluff.with({
		extra_args = { "--dialect", "postgres" } -- change to your dialect
	})
}]],
```

I think this would be a little more readable if we used brackets (not picky about the formatting otherwise).

Reply:
hm... I should have thought about this too, and now I'm not sure why not :D 

Anyways, it's changed now.